presto-main/src/main/java/com/facebook/presto/operator/PipelineContext.java

```diff
@@ -201,9 +202,9 @@ public void driverFinished(DriverContext driverContext)
 
         // merge the operator stats into the operator summary
         List<OperatorStats> operators = driverStats.getOperatorStats();
-        for (OperatorStats operator : operators) {
-            operatorSummaries.compute(operator.getOperatorId(), (operatorId, summaryStats) -> summaryStats == null ? operator : summaryStats.add(operator));
-        }
+        operators.stream().collect(Collectors.groupingBy(OperatorStats::getOperatorId))
```

Comment:
nit: static import `groupingBy`. Consider using the same approach with `merge` + `toImmutableList`